arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local raster/README.md

```diff
@@ -0,0 +1,61 @@
+# Create terrain from a local raster
+
+Set the terrain surface with elevation described by a raster file.
+
+![Create terrain from a local raster](create-terrain-from-a-local-raster.png)
+
+## Use case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. Supported raster formats include:
+
+* ASRP/USRP
+* CIB1, 5, 10
+* DTED0, 1, 2
+* GeoTIFF
+* HFA
+* HRE
+* IMG
+* JPEG
+* JPEG 2000
+* NITF
+* PNG
+* RPF
+* SRTM1, 2
+
+## How it works
+
+1. Create an `AGSScene` and add it to a `SceneView`.
+2. Create a `AGSRasterElevationSource` with a list of raster file paths.
+3. Add this source to the scene's base surface: `elevationSources.append(rasterElevationSource)`.
+
+## Relevant API
+
+* AGSRasterElevationSource
+* AGSSurface
+
+## About the data
+
+This raster data comes from Monterey, California.
+
+## Offline data
+
+1. Download the data from [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=98092369c4ae4d549bbbd45dba993ebc).
```

Comment:
Does the user have to download the data this way for this sample? I have seen samples where this is done automatically.